openshift/cd-applications/examples/acm-operator.yml

```diff
@@ -0,0 +1,21 @@
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: acm-operator
+  namespace: argocd
+spec:
+  destination:
+    server: 'https://api.lab02.gpslab.club:6443'
+  project: ocp-day2
+  source:
+    path: 10-acm-operator/
+    repoURL: >-
+      ssh://git@gitlab.consulting.redhat.com:2222/gps-laboratory/k8s-resources.git
```

Comment:
If possible, this should be included in this public repository on github or added to another public repository.
That repo is private